mediator/med.F90

```diff
@@ -763,6 +763,14 @@ subroutine InitializeIPDv03p1(gcomp, importState, exportState, clock, rc)
        if (ChkErr(rc,__LINE__,u_FILE_u)) return
     end do
 
+    ! Determine aoflux grid
+    call NUOPC_CompAttributeGet(gcomp, name='aoflux_grid', value=cvalue, isPresent=isPresent, isSet=isSet, rc=rc)
+    if (chkerr(rc,__LINE__,u_FILE_u)) return
+    if (.not. isPresent .and. .not. isSet) then
+       cvalue = 'ogrid'
```

Comment:
Is this the default value that is used in the current application? What do we need to set `aoflux_grid` for exchange grid implementation?

Reply:
I think there are three option for it: xgrid, agrid and ogrid